internal/cli/command/controlplane/kind.go

```diff
@@ -22,25 +22,40 @@ import (
 	"os"
 	"os/exec"
 	"path/filepath"
+	"regexp"
 	"runtime"
 
 	"emperror.dev/errors"
 	"github.com/banzaicloud/banzai-cli/internal/cli"
 	"github.com/banzaicloud/banzai-cli/internal/cli/input"
-	"github.com/ghodss/yaml"
 	log "github.com/sirupsen/logrus"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	kind "sigs.k8s.io/kind/pkg/cluster/config/v1alpha3"
-	"sigs.k8s.io/kind/pkg/container/cri"
+	yaml "gopkg.in/yaml.v3"
+	kind "sigs.k8s.io/kind/pkg/apis/config/v1alpha3"
 )
 
-const version = "v0.4.0"
+const version = "v0.8.1"
 const clusterName = "banzai"
 const kindCmd = "kind"
 
 func isKINDInstalled(banzaiCli cli.Cli) bool {
 	path, err := findKINDPath(banzaiCli)
-	return path != "" && err == nil
+	if path != "" && err == nil {
+		versionOutput, err := exec.Command(path, "version").CombinedOutput()
+		if err != nil {
+			return false
+		}
+
+		semver := regexp.MustCompile("v[0-9]+.[0-9]+.[0-9]+")
```

Comment:
have you considered https://github.com/Masterminds/semver or similar to check for a range instead of a specific version?

Reply:
Unfortunately, it won't work here now, since I have to parse the server out from a version output line.